.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Attempted to use the styles already prominent in the code base, to make contributing follow style guide by default.

Reply:
Sounds good. Does the current code follow those specs in all cases?  I know we do something a little different with switch/case where we out-dent the case statements.

Reply:
Yep, split out :+1: